backend/inspirehep/records/serializers/templates/latex_template.tex

```diff
@@ -45,3 +45,25 @@
 \VAR{- '\n[arXiv:' ~ data.arxiv_eprints.0.value ~ ' [' ~ data.arxiv_eprints.0.categories.0 ~ ']].' if 'arxiv_eprints' in data and 'categories' in data.arxiv_eprints.0}
 \VAR{- '\n' ~ data.report_numbers.0.value ~ '.' if 'dois' not in data and 'arxiv_eprints' not in data and 'report_numbers' in data }
 \VAR{- '\n%' ~ data.citations ~ ' citations counted in INSPIRE as of ' ~ data.today if data.citations != 0 }
+{%- if data.notes -%}
+    \VAR{ '\n [' }
+    {%- for note in data.notes -%}
+            \VAR{ 'erratum: ' -}
+             {%- if 'journal_title' in note -%}
+                \VAR{ note.journal_title ~ ',' }
+            {%- endif -%}
+             {%- if 'journal_issue' in note -%}
+                \VAR{- ' no.' ~  note.journal_issue ~ ', ' }
+            {%- endif -%}
+            {%- if 'page_range' in data.publication_info -%}
+                \VAR{ ' ' ~ note.page_range }
+            {%- elif 'artid' in note -%}
+                \VAR{ note.artid }
+            {%- endif -%}
+            \VAR{- ' (' ~ note.year ~ ')' if 'year' in note }
```

Comment:
you're not respecting the LaTeX US/EU distinction here. Maybe you should make L24-42 into a [macro](https://jinja.palletsprojects.com/en/2.11.x/templates/#macros) and reuse it here?